e2e/pages/nav-bar-page.js

```diff
@@ -0,0 +1,15 @@
+export class NavBarPage {
+    logo;
+    header;
+    githubRepoName
+    constructor(page) {
+        this.page = page;
+        this.logo = this.page.getByTestId('ng-hotrod-logo');
+        this.header = this.page.getByTestId('header');
+        this.githubRepoName = this.page.getByRole('link', { name: 'angularjs-material-vite' })
```

Comment:
GitrHub repo name after erdirect
<img width="297" alt="image" src="https://user-images.githubusercontent.com/5073455/220626970-eb466e94-9794-4dd2-966f-f88b79a9951a.png">

Reply:
Yes, but what part of the elements does it match against? The `<a>` element's text content?

Reply:
Yes, it referred to the <a> element's text content. 
Removed this Locator since it's not needed anymore - https://github.com/xlts-dev/angularjs-material-vite/pull/2#discussion_r1129374475

Reply:
Thx for clarifying :+1: